test/unit-tests/index.test.ts

```diff
@@ -709,30 +705,6 @@ describe("Custom Domain Plugin", () => {
 
         it("Get a given certificate name", async () => {
             AWS.mock("ACM", "listCertificates", certTestData);
-            AWS.mock("ACM", "describeCertificate", (params, callback) => {
-                const fifteenDaysAsMs = 15 * 24 * 60 * 60 * 1000;
-                const fifteenDaysInFuture = new Date(Date.now() + fifteenDaysAsMs);
-                const fifteenDaysInPast = new Date(Date.now() - fifteenDaysAsMs);
-                if (params.CertificateArn === "expired_cert_name") {
-                    callback(null, {
-                        Certificate: {
-                            CertificateArn: "doesnt_matter_wont_be_used",
-                            NotAfter: fifteenDaysInPast,
-                        }
-                    });
-                    return;
-                }
-                if (params.CertificateArn === "test_given_cert_name") {
-                    callback(null, {
-                        Certificate: {
-                            CertificateArn: params.CertificateArn,
-                            NotAfter: fifteenDaysInFuture,
-                        }
-                    });
-                    return;
-                }
-                throw new Error("Programmer error: didn't add new test mock data");
-            });
```

Comment:
don't need this because it was related to the separate API call to check expiries.